js/data.js

```diff
@@ -525,28 +526,28 @@ var FATERATOR = (function(fateratorModule) {
             // We're using the items array, so 
             // let's import that from our DATARATOR
             var beliefs = fateratorModule.DATA.aspects.beliefs;
-            returnAspect = beliefs.splice(fateratorModule.randomNum(beliefs.length), 1)[0];
+            returnAspect = beliefs[fateratorModule.randomNum(beliefs.length)];
```

Comment:
THIS was the race condition. I was removing things from the aspect array then depending on them. DUH IDIOT